src/Phinx/Db/Table.php

```diff
@@ -291,12 +293,12 @@ public function reset(): void
      * Valid options can be: limit, default, null, precision or scale.
      *
      * @param string|\Phinx\Db\Table\Column $columnName Column Name
-     * @param string|\Phinx\Util\Literal|null $type Column Type
+     * @param string|\Phinx\Util\Literal $type Column Type
      * @param array<string, mixed> $options Column Options
      * @throws \InvalidArgumentException
      * @return $this
      */
-    public function addColumn($columnName, $type = null, array $options = [])
+    public function addColumn(string|Column $columnName, string|Literal $type, array $options = [])
```

Comment:
This looks like it was possibly an accident? When supplying a `Column` instance in the first argument, it should already hold the type, right?

> Error: src/Table.php:67:47: PossiblyNullArgument: Argument 2 of Phinx\Db\Table::addColumn cannot be null, possibly null value provided (see https://psalm.dev/078)


https://github.com/cakephp/migrations/actions/runs/6269091941/job/17025092704?pr=628

Reply:
Migrations support for phinx 0.15 is here https://github.com/cakephp/migrations/pull/638